src/plugins/workspace/public/utils.ts

```diff
@@ -239,6 +246,46 @@ export const getDataSourcesList = (
     });
 };
 
+export const getDirectQueryConnections = async (dataSourceId: string, http: HttpSetup) => {
```

Comment:
Can we just reuse the function from DQC plugin?

Reply:
The logic in DQC is coupled within the component（`src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_data_connection/manage_direct_query_data_connections_table.tsx`）

Reply:
Nit: The best practice would be extract the logic to a util function, but I am OK that we leave it.